index.php

```diff
@@ -30,7 +30,7 @@
       }
       $rootPath = realpath("./client-conf/$conf_dir");
 
-      // Initialize archive object
+      // Initialize archive object ;;;; why doing this every time the user logs in, when the cert is static?
```

Comment:
I don't understand

Reply:
what I meant is: every user was getting the same static data archive as he wants to download his certs. There was no variation. So I was just wondering why the zip archive is created every time the user wants to download his files. Because before the  interface was able to edit the config files this was a redundant action (if I did not miss something). Now that the config files may change (when the admin edits the configs), zipping it again and again every time a user requests his files makes sense. 
I just forgot to remove the comment. You can remove it and ignore it.

Reply:
Ok thanks :) And yes you were right!